File: matiasmasso/Mat.NET/DT2/DTOCarrefourItem.vb
Public Class DTOCarrefourItem
    Property SupplierCode As String = "MATIA"
    Property Section As Integer = 61 'seccion de bebé
    Property Implantation As String = "610087"
    Property MadeIn As String
    Property SkuDsc As String
    Property SkuCustomRef As String
    Property SkuColor As String
    Property UnitsPerMasterBox As Integer
    Property UnitsPerInnerBox As Integer
    Property MasterBarCode As String
    Property Width As Integer
    Property Height As Integer
    Property Length As Integer
    Property Weight As Integer
    Property Albaran As String
    Property Linea As Integer

    Public Function Dimensions() As String
        Dim retval As String = String.Format("{0} x {1} x {2} cm", _Length / 10, _Width / 10, _Height / 10)
        Return retval
    End Function
End Class

File: matiasmasso/Mat.NET/DTO/Integracions/DTOCarrefourItem.cs
namespace DTO.Integracions
{

    public class DTOCarrefourItem
    {
        public string SupplierCode { get; set; } = "MATIA";
        public int Section { get; set; } = 61; // seccion de bebé
        public string Implantation { get; set; } = "610087";
        public string MadeIn { get; set; }
        public string SkuDsc { get; set; }
        public string SkuCustomRef { get; set; }
        public string SkuColor { get; set; }
        public int UnitsPerMasterBox { get; set; }
        public int UnitsPerInnerBox { get; set; }
        public string MasterBarCode { get; set; }
        public int Width { get; set; }
        public int Height { get; set; }
        public int Length { get; set; }
        public int Weight { get; set; }
        public string Albaran { get; set; }
        public int Linea { get; set; }

        public string Dimensions()
        {
            string retval = string.Format("{0} x {1} x {2} cm", Length / (double)10, Width / (double)10, Height / (double)10);
            return retval;
        }
    }

}
